src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs

```diff
@@ -39,6 +39,15 @@ public class DurableTaskOptions
         /// <value>A positive integer configured by the host. The default value is <c>32</c>.</value>
         public int ControlQueueBatchSize { get; set; } = 32;
 
+        /// <summary>
+        /// Gets or set the number of control queue messages that can be buffered in memory
+        /// at a time, at which point the dispatcher will wait before dequeuing any additional
+        /// messages. The default is 64.
+        /// </summary>
+        /// <remarks>This has historically always been fixed to 64, but increasing it may increase
+        /// throughput.</remarks>
+        public int ControlQueueBufferThreshold { get; set; } = 64;

```

Comment:
In V2 this is defaulted to 256. I suggest we do the same here. Also, consider copy/pasting my remarks there here: https://github.com/Azure/azure-functions-durable-extension/blob/ee7bc391671188268c005ba94aef4fa3e0dae44a/src/WebJobs.Extensions.DurableTask/Options/AzureStorageOptions.cs#L54-L65